NAMESPACE

```diff
@@ -1,4 +1,4 @@
-# Generated by roxygen2 (4.1.1): do not edit by hand
+# Generated by roxygen2 (4.1.1.9000): do not edit by hand
```

Comment:
Could you please document with released roxygen2? (I know this is a pain and the next version of roxygen will fix it)